mri/test/test_fourier_adjoint.py

```diff
@@ -26,13 +27,14 @@ def setUp(self):
         """
         self.N = 64
         self.max_iter = 10
+        self.num_channels = [1, 16]
```

Comment:
For the test `num_channels = [1, 16]` is probably too much, instead, you could reduce it to `[1, 2]`

Reply:
Sure.